### ISSUE TITLE
[FLATPAK] Game starts stuttering after some time of playing

### ISSUE THREAD

**Issue by moeraccoon** [goatcorp/FFXIVQuickLauncher#1302]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to the game updating and plugins not working correctly.

### What did you do?

It's worth noting that I play on linux. With that in mind, the game runs flawlessly when I start from the official launcher.

Initially I didn't start the launcher properly, I opened a thread here for support a couple of weeks ago.
However when I launcher the launcher incorrectly the game worked flawlessly until I had to exit, when I did that the process kept running. Plugins worked, game ran fine.

After fixing the setup and fixing that issue another problem showed up, the one I'm posting for: When I start playing things are fine. As time goes I eventually get some pretty noticeable stuttering. It's most noticeable when I turn around, wither with keyboard or mouse.

I do not believe the plugins are to blame cause I tried disabling them.

I found online some people suggesting to add a file in the game folder with a line. The file is named dxvk.conf and inside there is a single line:
dxgi.maxFrameLatency = 1

It did not seem to help.

### Platform

Windows

### Wine/Proton runner version

The launcher handles it, I left it as default

**Comment by ChibiChoko** [comment]
I've been having the same issue using the AUR version. After a certain duration or number of area changes, panning the camera starts jittering. It's even more pronounced when moving while panning the camera.

**Comment by moeraccoon** [comment]
You only get it when using the launcher too? Doesn't happen with the official launcher?

**Comment by PaddyMD** [comment]
I'm getting the same issue on Linux. The games runs perfectly fine for 1 hour, then a huge amount of stuttering starts. It's seemingly triggered by mouse & keyboards inputs to a degree. Issue is only present when using the XIVlauncher. No such issue with the official launcher.

**Comment by eduardomcv** [comment]
Same here. Switched back to default launcher using GE-Proton and the game works flawlessly.

**Comment by M-Paulsen** [comment]
Same issue. Bad stuttering when playing, after some time or maybe some specific action. OpenSUSE Tumbleweed, all updated.

I tried idling in Limsa for about 90 minutes, no issues. Traveled to a different place, still no issues. Entered dungeon, lag when moving camera with mouse. Restart game, immediately enter same dungeon, no problem. So there is some combination of time and what you're doing in play here?

I've tried pointing XIVLauncher at another wine install (proton-ge-wine), with no change. I used Lutris before (but that won't launch the game at all now, hence using XIVLauncher), and never had this issue.

I'm going to do some more testing, though it's quite annoying having to wait an hour every time. :)

**Comment by zee5679** [comment]
Can report same issue about after an hour any input tanks the fps greatly

**Comment by ChibiChoko** [comment]
Has anyone tried turning off FSync and using ESync instead? i tried that and haven't run into this issue again.

**Comment by rankynbass** [comment]
Give this wine version a try. It includes a patch ported over from Valve's proton-wine that fixes a problem very much like this issue. (stuttering after 40+ minutes).

https://github.com/goatcorp/wine-xiv-git/releases/tag/8.5.r4.g4211bac7

**Comment by moeraccoon** [comment]
Sorry for my ignorance but how do I use it on the launcher?

**Comment by moeraccoon** [comment]
So I got it to run using that version of steam that has been linked.
No improvement.

**Comment by moeraccoon** [comment]
I'd also like to add something. I bought a new ssd and a new GPU, so I thought I'd reinstall and while I was at it I tried another distro.
Different distro AND different GPU and I still get that problem. I would've bet the new gpu would be enough but alas.

**Comment by ChibiChoko** [comment]
So I've been testing this more and every instance of this stuttering only happens when I launch XIVlauncher via steam. If I launch it externally, I never run into this issue.

**Comment by moeraccoon** [comment]
> So I've been testing this more and every instance of this stuttering only happens when I launch XIVlauncher via steam. If I launch it externally, I never run into this issue.

Yes, I had the same happening to me, I think I mentioned it in my OP.
In another thread a fix was found, the reason why it doesn't run correctly from steam is because the guide was written with steamdeck in mind and the launch options were as well.
Delete your launch options and replace them with this:

LD_PRELOAD="" %command% run --branch=master --arch=x86_64 --command=xivlauncher dev.goats.xivlauncher

It should work.

**Comment by Horcrux94** [comment]
This should be closed now, right? 1.0.4.0 should have shipped a new wine without this problem.

**Comment by moeraccoon** [comment]
The problem wasn't the wine version, at least in my case. 
The problem was that non steamdeck users were not supposed to use the full line posted in the guide by the devs under launch options and, instead, use the following:
LD_PRELOAD="" XL_SECRET_PROVIDER=FILE %command% run --parent-expose-pids --parent-share-pids --parent-pid=1 --branch=stable --arch=x86_64 --command=xivlauncher dev.goats.xivlauncher

I had to reinstall my system yesterday and still had to check my github posts here to make sure I used the right line because the guide doesn't give this bit of info.
